CMakeLists.txt

```diff
@@ -28,9 +28,9 @@ catkin_package(
 
 include_directories(
   include
-  ${Boost_INCLUDE_DIRS}
-  ${OpenCV_INCLUDE_DIRS}
+#  ${ASSIMP_INCLUDE_DIRS}
```

Comment:
I would like to keep it here, because it should be added. But the current `ASSIMP_INCLUDE_DIRS` is broken. It is fixed by https://github.com/assimp/assimp/pull/2161, but not released on ubuntu 18.04. So we need to wait for 20.04 for the fix.

But might be good to add a comment about this in the file.